SPECS/kernel-azure/config

```diff
@@ -512,6 +512,7 @@ CONFIG_CPU_SRSO=y
 # CONFIG_GDS_FORCE_MITIGATION is not set
 CONFIG_MITIGATION_RFDS=y
 CONFIG_MITIGATION_SPECTRE_BHI=y
+CONFIG_MITIGATION_ITS=y
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/858073be8899e16cc469be9c1837d6c3a64887fa